cwl_wes/custom_config.py

```diff
@@ -70,12 +263,69 @@ class ServiceInfoConfig(FOCABaseConfig):
 
 
 class TesServerConfig(FOCABaseConfig):
+    """Model for tes server configuration.
+
+    Args:
+        url: TES Endpoint URL.
+        timeout: Request time out.
+        status_query_params: Request query parameters.
+
+    Attributes:
+        url: TES Endpoint URL.
+        timeout: Request time out.
+        status_query_params: Request query parameters.
+
+    Raises:
+        pydantic.ValidationError: The class was instantianted with an illegal
+            data type.
+
+    Example:
+        >>> TesServerConfig(
+        ...     url='https://tes.endpoint',
+        ...     timeout=5,
+        ...     status_query_params='FULL'
+        ... )
+        TesServerConfig(url='https://tes.endpoint', timeout=5, status_query_params='FULL')
+    """
     url: str = 'https://csc-tesk.c03.k8s-popup.csc.fi/'
     timeout: int = 5
     status_query_params: str = 'FULL'
 
 
 class DRSServerConfig(FOCABaseConfig):
```

Comment:
I think this should be part of the `EndpointConfig`, and so should `IdConfig`. Perhaps it's also better/clearer to rename this to `ControllerConfig`